packages/handlersjs-storage/tsconfig.spec.json

```
@@ -0,0 +1,26 @@
+{
+  "extends": "@digita-ai/typescript-config/tsconfig.spec.json",
+  "compilerOptions": {
+    "baseUrl": "./lib",
+    "outDir": "./dist",
+    "module": "commonjs",
+    //
+    "types": [
```

same here, or is this intentional?
```suggestion
    "types": [
```